### ISSUE TITLE
Unable to save Spanish version of page

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#16538]
1. Open the page properties of the `about us` page
2. Pull down the Espanol from the language picker and say `yes` to copy properties
3. Make no changes and hit save.  It throws:

```
:40:25.522  INFO  util.ActivityLogger - class com.dotcms.content.elasticsearch.business.ESContentletAPIImpl: demo.dotcms.com : Content Unlocked , StartDate: N/D; EndDate: N/D; User:dotcms.org.1; ContentIdentifier: c12fe7e6-d338-49d5-973b-2d974d57015b
11:40:25.861  INFO  reindex.ReindexThread - -----------
11:40:25.861  INFO  reindex.ReindexThread - Total Indexed        : 8446
11:40:25.861  INFO  reindex.ReindexThread - ReindexEntries found : 1
11:40:25.861  INFO  reindex.ReindexThread - BulkRequests created : 2
11:40:36.865  WARN  web.ContentletWebAPIImpl - java.lang.NullPointerException
java.lang.NullPointerException: null
	at com.dotmarketing.portlets.contentlet.business.web.ContentletWebAPIImpl.validateNewContentPage(ContentletWebAPIImpl.java:630) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.business.web.ContentletWebAPIImpl.saveWebAsset(ContentletWebAPIImpl.java:500) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.business.web.ContentletWebAPIImpl.saveContent_aroundBody0(ContentletWebAPIImpl.java:158) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.business.web.ContentletWebAPIImpl$AjcClosure1.run(ContentletWebAPIImpl.java:1) ~[classes/:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[classes/:?]
	at com.dotmarketing.db.LocalTransaction.wrapReturnWithListeners(LocalTransaction.java:40) ~[classes/:?]
	at com.dotcms.aspects.interceptors.WrapInTransactionMethodInterceptor.invoke(WrapInTransactionMethodInterceptor.java:23) ~[classes/:?]
	at com.dotcms.aspects.aspectj.WrapInTransactionAspect.invoke(WrapInTransactionAspect.java:41) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.business.web.ContentletWebAPIImpl.saveContent(ContentletWebAPIImpl.java:132) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.ajax.ContentletAjax.saveContent_aroundBody14(ContentletAjax.java:1556) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.ajax.ContentletAjax$AjcClosure15.run(ContentletAjax.java:1) ~[classes/:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.8.10.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[classes/:?]
	at com.dotcms.aspects.interceptors.CloseDBMethodInterceptor.invoke(CloseDBMethodInterceptor.java:34) ~[classes/:?]
	at com.dotcms.aspects.aspectj.CloseDBAspect.invoke(CloseDBAspect.java:42) ~[classes/:?]
	at com.dotmarketing.portlets.contentlet.ajax.ContentletAjax.saveContent(ContentletAjax.java:1380) ~[classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule$1.doFilter(CreatorModule.java:229) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.CreatorModule.executeMethod(CreatorModule.java:241) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.DefaultRemoter.execute(DefaultRemoter.java:379) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.impl.DefaultRemoter.execute(DefaultRemoter.java:332) ~[dot.dwr-3rc2modified_3.jar:?]
	at com.dotcms.repackage.org.directwebremoting.dwrp.BaseCallHandler.handle(BaseCallHandler.java:104) ~[dot.dwr-3rc2modified_3.jar:?]
```

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/16542

**Comment by bryanboza** [comment]
After the last changes, when you add a new version of any page in Spanish or any other language, we are leaving setting the session language too, then if you try to upload multiple files we upload those files on this language too, not in the default. 
To reproduce: 
- Login 
- Go to site browser --> about us and create the Spanish version of the index page 
- Upload multiple files 

We are creating those files in Spanish

**Comment by bryanboza** [comment]
Fixed, the main problem was fixed, new enhancement here #16555
